arc/cli/loading_interface.py

```
@@ -0,0 +1,329 @@
+        return f"{minutes:02d}:{seconds:02d}"
+
+
+class StreamingResultsDisplay:
+    """Display streaming results as they arrive."""
+    
+    def __init__(self, console: Console):
+        """Initialize with console for output."""
+        self.console = console
+        self.results_buffer = []
+        
+    def add_result(self, result: Dict[str, Any]) -> None:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)